### PR TITLE
Basic backend with one characteristic

### PR DESCRIPTION
**Pull request by g-ortuno**
Please review @jyasskin @scheib

### CODE REVIEW

**Comment by jyasskin** [comment]
Can you make your commit message more descriptive? In particular, this is a Heart Rate Service that exposes one of its characteristics.

File: app/src/main/java/io/github/webbluetoothcg/bletestperipheral/HeartRateServiceFragment.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
+package io.github.webbluetoothcg.bletestperipheral;
+
+
```
Comment (jyasskin) [inline app/src/main/java/io/github/webbluetoothcg/bletestperipheral/HeartRateServiceFragment.java]: Avoid double-blank-lines. (https://google-styleguide.googlecode.com/svn/trunk/javaguide.html#s3-source-file-structure)

File: app/src/main/java/io/github/webbluetoothcg/bletestperipheral/HeartRateServiceFragment.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import java.util.UUID;
+
+public class HeartRateServiceFragment extends ServiceFragment {
+
+  ///////////////////////
+  ////// Constants //////
```
Comment (jyasskin) [inline app/src/main/java/io/github/webbluetoothcg/bletestperipheral/HeartRateServiceFragment.java]: Don't enclose comments in boxes: https://google-styleguide.googlecode.com/svn/trunk/javaguide.html#s4.8.6.1-block-comment-style

File: app/src/main/java/io/github/webbluetoothcg/bletestperipheral/HeartRateServiceFragment.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  ///////////////////////
+  private static final UUID HEART_RATE_SERVICE_UUID = UUID
+      .fromString("0000180D-0000-1000-8000-00805f9b34fb");
+
+  private static final UUID BODY_SENSOR_LOCATION_UUID = UUID
```
Comment (jyasskin) [inline app/src/main/java/io/github/webbluetoothcg/bletestperipheral/HeartRateServiceFragment.java]: Maybe link to the definitions of these services and characteristics? https://developer.bluetooth.org/gatt/services/Pages/ServiceViewer.aspx?u=org.bluetooth.service.heart_rate.xml and https://developer.bluetooth.org/gatt/characteristics/Pages/CharacteristicViewer.aspx?u=org.bluetooth.characteristic.body_sensor_location.xml.

File: app/src/main/res/layout/fragment_heart_rate.xml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (jyasskin) [inline app/src/main/res/layout/fragment_heart_rate.xml]: Is `FrameLayout` right for the fragment's layout? http://developer.android.com/training/basics/fragments/fragment-ui.html seems to say it's good for the Activity's layout that holds fragments, but I'd expect the fragment to have several children.

**Comment by jyasskin** [comment]
LGTM after the above comments.

File: app/src/main/java/io/github/webbluetoothcg/bletestperipheral/HeartRateServiceFragment.java (hunk 1 above)
Comment (g-ortuno) [inline app/src/main/java/io/github/webbluetoothcg/bletestperipheral/HeartRateServiceFragment.java]: Done.

File: app/src/main/java/io/github/webbluetoothcg/bletestperipheral/HeartRateServiceFragment.java (hunk 2 above)
Comment (g-ortuno) [inline app/src/main/java/io/github/webbluetoothcg/bletestperipheral/HeartRateServiceFragment.java]: Done.

File: app/src/main/java/io/github/webbluetoothcg/bletestperipheral/HeartRateServiceFragment.java (hunk 3 above)
Comment (g-ortuno) [inline app/src/main/java/io/github/webbluetoothcg/bletestperipheral/HeartRateServiceFragment.java]: Done.

File: app/src/main/res/layout/fragment_heart_rate.xml (hunk 4 above)
Comment (g-ortuno) [inline app/src/main/res/layout/fragment_heart_rate.xml]: Done. The code was auto generated so I didn't pay that much attention to it. I changed it to RelativeLayout which makes more sense.

**Pull request merged**